docker/get-tmp.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I think this file can safely be deleted, across all these PRs. This must have been accidentally committed in the base repo, I'll have to clean that up at some point.